podvm/qcow2/rhel/qemu-rhel.pkr.hcl

```diff
@@ -50,30 +50,31 @@ build {
   }
 
   provisioner "file" {
-    source      = "qcow2/misc-settings.sh"
-    destination = "~/misc-settings.sh"
+    source      = "qcow2/selinux_relabel.sh"
+    destination = "~/selinux_relabel.sh"
   }
 
   provisioner "shell" {
     remote_folder = "~"
-    environment_vars = [
-        "CLOUD_PROVIDER=${var.cloud_provider}",
-        "PODVM_DISTRO=${var.podvm_distro}",
-	]
     inline = [
-      "sudo -E bash ~/misc-settings.sh"
+      "sudo bash ~/selinux_relabel.sh"
     ]
   }
 
   provisioner "file" {
-    source      = "qcow2/selinux_relabel.sh"
-    destination = "~/selinux_relabel.sh"
+    source      = "qcow2/misc-settings.sh"
+    destination = "~/misc-settings.sh"
   }
 
   provisioner "shell" {
     remote_folder = "~"
+    environment_vars = [
+        "CLOUD_PROVIDER=${var.cloud_provider}",
+        "PODVM_DISTRO=${var.podvm_distro}",
+	]
     inline = [
-      "sudo bash ~/selinux_relabel.sh"
+      "sudo -E bash ~/misc-settings.sh"
     ]
   }
+
```

Comment:
extra line? also maybe add a comment in misc_settings that it must be after the relabeling for the future.. your call, LGTM overall, thanks!

Reply:
Thanks for the review, both fixed!